src/librustdoc/html/render.rs

```diff
@@ -866,15 +866,9 @@ fn write_shared(cx: &Context,
     write(cx.dst.join("main.css"),
           include_bytes!("static/styles/main.css"))?;
     if let Some(ref css) = cx.shared.css_file_extension {
-        let mut content = String::new();
-        let css = css.as_path();
-        let mut f = try_err!(File::open(css), css);
-
-        try_err!(f.read_to_string(&mut content), css);
+        let content = try_err!(fs::read(&css), &css);
```

Comment:
This should be `read_string`, right?

Reply:
The original used `read_to_string` but there's no reason to create a string here; it's just copying the contents to a new file.  Actually, this should use `fs::copy`...

Reply:
Fixed.